Animations/Animation.js

```diff
@@ -1,31 +1,16 @@
 'use strict';
 
-var React = require("react/addons");
+var React = require('react/addons');
 var Arrival = require('Arrival');
-var Easings = require("./AnimationEasings.js");
-var ReactTransitionGroup = React.addons.TransitionGroup;
+var Easings = require('./AnimationEasings');
 
-//makeTransition FUNCTION: creates animation for wrapped component based on props of wrapper component
-  //@param displayName STRING: intended name of the wrapper animation component
+//makeAnimation FUNCTION: creates animation for wrapped component based on props of wrapper component
   //@param transitions OBJECT: properties that define the default animation properties for the animation component wrapper
-var makeTransition = function (displayName, transitions) {
-
-  // setAnimationDomNode FUNCTION: function to handle manipulating and setting a TransitionGroup in the DOM.
-    //@param mount STRING: the transition key you are looking to affect
-    //@param callback FUNCTION: callback attribute passed in from the containing function to act on function completion.
-  var setAnimationDomNode = function (mount, callback) {
-      this.el = this.getDOMNode();
-      this.$el = $(this.el);
-
-      // requestAnimationFrame FUNCTION: Calls the specified function updating an animation before the next browser repaint. Defined in window.
-      requestAnimationFrame(function () {
-        this.$el.css(transitions[mount]);
-        requestAnimationFrame(function () {
-          this.$el.css(transitions[mount + 'Active']);
-          Arrival(this.$el, callback);
-        }.bind(this));
-      }.bind(this));
-  };
+var makeAnimation = function (transitions) {
+  //If a second parameter is passed in as the desired class name for the animation, set this as className, othewise, use the animation's default class name
+  var className;
+  var args = Array.prototype.slice.call(arguments);
+  className = args.length === 1 ? transitions.className : args[1];
```

Comment:
Rather than slicing the arguments array and doing that work, could we instead just have another input called `customClassName` for example and, if `customClassName` is defined set `className` equal to `customClassName` on line 13.